.github/workflows/Specs.yml

```diff
@@ -0,0 +1,41 @@
+name: Specs
+
+jobs:
+  specs:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04]
+        ruby: [2.6.3]
+
+    name: ${{ matrix.os }} / Ruby ${{ matrix.ruby }}
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout git
+        uses: actions/checkout@v1
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        if: ${{ matrix.ruby != 'system' }}
+        with:
+          ruby-version: ${{ matrix.ruby }}
+
+      - name: Install packages
+        run: sudo apt-get install postgresql-server-dev-9.4        
```

Comment:
`sudo` should be fine, see https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners and https://stackoverflow.com/questions/57982945/how-to-apt-get-install-in-a-github-actions-workflow